src/test/ui/rfcs/rfc-2396-target_feature-safe-calls.rs

```diff
@@ -0,0 +1,19 @@
+#[target_feature(enable = "sse2")]
+fn foo() {}
+
+#[target_feature(enable = "sse2")]
+fn bar() {
+    foo();
+}
+
+#[target_feature(enable = "avx")]
```

Comment:
Let's also do a case of more than just one where the caller has one, missing one.